src/components/form-elements/Input.jsx

```diff
@@ -53,28 +53,67 @@ export const TYPE = Object.freeze({
 });
 
 export const SIZE = Object.freeze({
-  SMALL: 'small',
   LARGE: 'large',
   NORMAL: 'normal',
 });
 
 export const COLOR = Object.freeze({
-  NORMAL: 'normal',
-  LIGHT: 'light',
-  LIGHT_ALT: 'light-alt',
+  DEFAULT: 'default',
+  WHITE: 'white',
 });
 
 export type InputPropsType = {
+  /**
+   * Optional specification for input type
+   * @example <Input type="email" placeholder="placeholder" />
+   */
   type?: ?InputType,
+  /**
+   * Optional specification for input value, which can be string or number
+   * @example <Input type="email" value={2} />
+   */
   value?: string | number,
+  /**
+   * There are two sizes options for most of the form elements
+   * @example <Input size="normal" placeholder="placeholder" />
+   * @see size="normal" https://styleguide.brainly.com/latest/docs/interactive.html?size="normal"#input
+   * @see size="large" https://styleguide.brainly.com/latest/docs/interactive.html?size="large"#input
+   */
   size?: ?InputSizeType,
+  /**
+   * There are two color variants for form elements, default does not have to be specified
+   * @example <Input color="white" placeholder="placeholder" />
+   * @see color="default" https://styleguide.brainly.com/latest/docs/interactive.html?color="default"#input
+   * @see color="white" https://styleguide.brainly.com/latest/docs/interactive.html?color="white"#input
+   */
   color?: ?InputColorType,
+  /**
+   * Optional additional boolean to specified if it's valid
```

Comment:
do we need both `Optional additional` next to each other?

Reply:
nope, we don't I will fix it. thx

Reply:
done.